### PR TITLE
[ZEPPELIN-5290] NPE on empty dynamic form input

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Simple PR to address NPE on empty dynamic form input. Instead of throwing NPE, just make the form value as empty string. 

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5290

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Review by cuspymd — APPROVED** [review]
Looks good!!

**Comment by zjffdu** [comment]
Will merge if no more comment

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java [hunk 1]
```diff
@@ -373,7 +373,11 @@ public static String getSimpleQuery(Map<String, Object> params, String script, b
         }
       } else {
         // single-selection
-        expanded = value.toString();
+        if (value == null) {
```
Comment (Reamer) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java:376]: [StringUtils.defaultString](http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#defaultString%28java.lang.String%29) looks more elegant.

Comment (zjffdu) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java]: Thanks for the review @Reamer  I have updated it

**Comment by Reamer** [comment]
LGTM

**Pull request closed without merging**